ops/terraform/services/common/s3.tf

```diff
@@ -29,27 +28,16 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "this" {
 }
 
 resource "aws_s3_bucket_logging" "this" {
-  # TODO: Make this work better for ephemeral environments, etc
   count = local.is_ephemeral_env ? 0 : 1
 
   bucket = aws_s3_bucket.this.id
 
-  # TODO: consider adding this...
-  # expected_bucket_owner = local.account_id
+  expected_bucket_owner = local.account_id
```

Comment:
This is a good change; but, while I do not expect we would be shipping logs across accounts, I'm not 100% certain we won't be. (ie. multiple aws accounts per environment with our upcoming v4/cloudtamer migration).

Reply:
Not requesting any changes here, just something to note.

Reply:
If you want, I can switch this back and we can circle back when we get to working on v4 migration.

Reply:
No it's ok. Just something to note during our migration. Doubt it will come up.